lib/core/enums/local_settings.dart

```diff
@@ -94,6 +94,8 @@ enum LocalSettings {
   postFabLongPressAction(name: 'settings_post_fab_long_press_action', label: ''),
   enableCommentNavigation(name: 'setting_enable_comment_navigation', label: 'Enable Comment Navigation Buttons'),
   combineNavAndFab(name: 'setting_combine_nav_and_fab', label: 'Combine FAB and Navigation Buttons'),
+
+  draftsCache(name: 'drafts_cache', label: ''),
```

Comment:
One thing to note about here is that we probably don't want to save the drafts that get created when we export/import the user settings to JSON!

Reply:
Good point! I added a mechanism to exclude settings from import/export, and I excluded drafts.